tests/examples/test_examples.py

```diff
@@ -58,10 +60,8 @@ def run_test(filepath):
     ]
 )
 def test_finetune_example(tmpdir, step, file):
-    with tmpdir.as_cwd():
```

Comment:
The path `tmpdir.as_cwd()` is all different in each test parametrized above, which let to downloading the same dataset to different places.